### ISSUE TITLE
Error if "movie.data.runtime" is NULL

### ISSUE THREAD

**Issue by DanCooper** [HristoKolev/TvDbSharper#30]
Looks like the case "movie.data.runtime = NULL" isn't handled:
```
Newtonsoft.Json.JsonSerializationException: Error converting value {null} to type 'System.Int32'. Path 'data.runtime', line 1, position 308.
```
To reproduce use TVDb Id 198054.

**Comment by HristoKolev** [comment]
Thanks. Will look at it.

**Comment by HristoKolev** [comment]
fixed in v4.0.9
